src/main/java/org/sonarjs/cli/report/JsonReporter.java

```diff
@@ -71,7 +71,7 @@ private void issueToJSON(Issue issue, Function<String, RuleDetails> ruleDescript
     writer.beginObject();
     ClientInputFile inputFile = issue.getInputFile();
     if (inputFile == null) {
-      LOGGER.warn("Issue without file : " + shortIssueDescription(issue));
+      LOGGER.error("Issue without file : " + shortIssueDescription(issue));
```

Comment:
@inverno you log but not output this message, right?

Reply:
@vilchik-elena it goes to stderr now, so it will be outputted, but not interfere with JSON parsing